apps/judicial-system/web/src/routes/Prosecutor/Overview/Overview.tsx

```diff
@@ -209,9 +209,15 @@ export const Overview: React.FC = () => {
               accusedName={workingCase.accusedName}
               accusedNationalId={workingCase.accusedNationalId}
               accusedAddress={workingCase.accusedAddress}
+              // defender info set by the prosecutor has priority until the case has been ruled on
               defender={{
-                name: workingCase.defenderName || '',
-                email: workingCase.defenderEmail,
+                name:
+                  workingCase.requestedDefenderName ||
```

Comment:
Isn't this only supposed to be the defender info, not the requested defender?

Reply:
Well, this is a bit tricky.

For a request that has been ruled on it makes sense to only use the defender. That is what is done in SignedVerdictOverview.

For requests that are still progress, this is a bit subtle.
1. If we don't use the requested defender, then no defender will be shown in InfoCard, until a judge has opened the page with the defender fields.
2. Every time a judge opens the page with the defender fields, the defender will be set to the requested defender.

So, I think it really makes sense to show the requested defender on the over view screens for both the prosecutor and the judge if the requested defender has been set and only show the defender if the requested defender has not been set. Otherwise, we will either not show any defender in InfoCard (defender has not been set) or we will show the defender, which will be overwritten by the requested defender when the judge enters the page with the defender fields.

Pretty complicated, right.

Reply:
Þessi breyting sem ég gerði er að koma í veg fyrir villun sem Baldur lýsir í Asana miðanu :)

Reply:
Flott :) Baldur segir að "Ef dómur hefur skipað annan verjanda, þá gildir það sem dómur skráði (sama hvað ákærandi skráði)" þannig að þetta ætti þá að vera `workingCase.defenderName || workingCase.requestedDefenderName`, er það ekki?

Reply:
Það dugar ekki til, því það tekur eingögnu á birtingunni.

Í hvert skipti sem dómari fer á skjáinn "Fyrirtaka", þá er verjandi sem dómari setti inn, yfirskrifaður með verjandanum sem saksóknari setti inn. Það væri því villandi að birta það sem dómarinn skráði, nema þegar búið er að úrskurða eða saksóknari valdi ekkert sjáfur.